Update 12 raw data analysis (#46)
jessegmeyerlab/proteomics-tutorial: content/12.raw-data-analysis.md

### Quality control

Quality control should be a central aspect of any mass spectrometry-based study to ensure reproducibility of generated results.
There are two types of quality controls that can be conducted for any kind of mass spectrometry experiment.
The first one is focused on monitoring the perfomance of the instruments themselves (e.g. HPLC and mass spectrometer), whereas the second one is focused on your experiments. 
For further reading, we recommend to take a look at issue 11 on quality control published in the journal _Proteomics_ in 2011 [@DOI:10.1002/pmic.201190020], especially the review by Köcher _et al._ [@DOI:10.1002/pmic.201000578], as well as the review published by Bittremieux _et al._ in 2017 [@DOI:10.1002/mas.21544].

#### Instrument Performance

It is generally advisable to monitor instrument performance regularly. 

> Instrument calibrations in regular intervals help ensure that performance is appropriate.

Reviewer: ```suggestion
Instrument calibrations in regular intervals help ensure that performance is maintained.
Often basic calibration and sensitivity can be checked by direct infusion of a standard.
```

> During the calibration you can check injection times and intensity of the ions of your calibration mix.

Reviewer: ```suggestion
During the calibration you can check injection times (for ion trap instruments) and intensity of the ions in the calibration mix.
```

> To find out if your mass spectrometer is performing as expected, you can analyze complex samples, such as tryptic digests of whole-cell lysates (e.g. HeLa cells, HEK cells, yeast, etc.) or tryptic digests of purified proteins.

Reviewer: ```suggestion

After ensuring good calibration and signal with the simple calibration mixture, it is advisable to analyze complex samples, such as tryptic digests of whole-cell lysates (e.g. HeLa cells, HEK cells, yeast, etc.) or tryptic digests of purified proteins.
The additional check with a complex sample ensures all aspects of the system are working together correctly, especially the liquid chromatography and emitter.
```

> These digests should be analyzed after every instrument calibration and if required in between samples when acquiring more extensive batches.

Reviewer: ```suggestion
These digests should be analyzed after every instrument calibration and periodically between samples when acquiring more extensive batches.
```

> Measured tryptic digests can be searched with software of your choice and the numbers of identified peptide precursors and proteins can be compared.

Reviewer: ```suggestion
Data measured from tryptic digests should be analyzed by the software of your choice and the numbers of identified peptide precursors and proteins can be compared with previous controls for consistency.
```

> Measurments of digested purified proteins are great for identifying retention time shifts and mass accuracy problems. 

Reviewer: ```suggestion

Another strategy is to analyze digested purified proteins, which easily enable discovery of retention time shifts and mass accuracy problems. 
```

In case you are working with a Thermo mass spectrometer, you can open the acquired .raw file directly either in FreeStyle or in Qual Browser and look for specific m/z values of your peptides.
Looking at the intensity of the extracted peaks will help identify sensitivity fluctuations. 

> Carry-over between different measurments can be identified from blank measurements which are subsequently analyzed with your search software of choice.

Reviewer: ```suggestion

Carry-over between different measurments can be identified from blank measurements which are subsequently analyzed with your search software of choice.
```

Blank measurements can be injections of different buffers, water or the starting conditions of your liquid chromatography. 
In case of increased detection of carry-over, injections with trifluoroethanol can be performed.

> Another factor to take into consideration is the stability of your electrospray.

Reviewer: ```suggestion

Another factor to take into consideration is the stability of your electrospray.
```

> Electrospray stability tends to worsen over time with column wear-out, as well as when measuring samples with residual contaminants, such as salts or detergents. 

Reviewer: ```suggestion
Electrospray stability tends to worsen over time as columns wear, as well as when measuring samples with residual contaminants, such as salts or detergents. 
```
